lib/models/builder.js

```diff
@@ -208,33 +227,41 @@ module.exports = CoreObject.extend({
     });
   },
 
+  _reportVizInfo: function (vizInfo) {
+    if (this.project && this.project.addons.length) {
+      this.project.addons.forEach(function (addon) {
+        if (addon._buildAnalytics) {
```

Comment:
Should use a private symbol (something like Embers symbol) for now, until we have time for an RFC.